prometheusbp/metrics_buckets_old.go

```diff
@@ -0,0 +1,14 @@
+//go:build !metrics_spec
```

Comment:
actually we probably want to be extra explicit with the build tags, and the Go ones are all one word things like netgo, so maybe:

```suggestion
//go:build oldprombuckets
```